compiler/natives/src/syscall/syscall_unix.go

```diff
@@ -67,6 +67,10 @@ func Syscall(trap, a1, a2, a3 uintptr) (r1, r2 uintptr, err Errno) {
 		return uintptr(array.Length()), 0, 0
 	}
 	if trap == SYS_EXIT {
+		process := js.Global.Get("process")
+		if process != js.Undefined {
```

Comment:
Style-wise, this is a good time to inline `process := js.Global.Get("process")` into the if statement:

```Go
if process := js.Global.Get("process"); process != js.Undefined {
	process.Call("exit", a1)
}
```

`process` is only needed inside the if statement, so it's good to give it a smaller scope.

(/cc @gmlewis FYI.)